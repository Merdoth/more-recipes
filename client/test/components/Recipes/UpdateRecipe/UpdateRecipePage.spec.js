import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { UpdateRecipePage } from
  '../../../../components/Recipes/UpdateRecipe/UpdateRecipePage.jsx';

  /* global jest */

const setup = () => {
  const props = {
    goToRecipes: () => jest.fn(),
  };
  return shallow(<UpdateRecipePage {...props} />);
};

describe('UpdateRecipePage Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
