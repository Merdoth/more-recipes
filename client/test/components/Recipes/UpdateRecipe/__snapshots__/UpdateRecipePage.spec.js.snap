// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateRecipePage Component snapshot it should render the right amount of elements 1`] = `
<div
  className="row deco"
>
  <div
    className="col-md-4 offset-md-4 cover1"
  >
    <div
      className="form-deco"
    >
      <Connect(UpdateRecipeForm)
        goToRecipes={[Function]}
      />
    </div>
  </div>
</div>
`;
